app/Providers/AuthServiceProvider.php

```diff
@@ -31,5 +38,20 @@ class AuthServiceProvider extends ServiceProvider
      */
     public function boot(): void {
         $this->registerPolicies();
+
+        Passport::useClientModel(OAuthClient::class);
+
+        // Override passport routes
+        Route::group(['prefix' => 'oauth', 'as' => 'oauth.'], function () {
```

Comment:
Can we move this to `routes/web.php`? Just so we have everything in one place.

Reply:
This has already been discussed [here](https://github.com/Traewelling/traewelling/pull/1304#discussion_r1089816858)
This has to be here, otherwise the routes do not get overwritten